media-libs/libvmaf/libvmaf-2.3.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit meson multilib-minimal
```

Comment:
You can use meson-multilib.

By the way, do we actually need multilib for this? Are there 32 bit applications which need it?

Reply:
> You can use meson-multilib.

Noted!

> By the way, do we actually need multilib for this? Are there 32 bit applications which need it?

Now that you bring it up... This entirely unrelated to x86, x86 don't need this right ? That's what I had in mind initially, but I am still learning about how to handle other arches. If it's not needed for x86 I don't think it's useful.

Reply:
Entirely unrelated, yep! And no worries. The different platforms we learn about are part of what makes Gentoo fun :)

multilib is specifically for (mainly) where we want to build something twice to get a 32 bit library and a 64 bit one for supporting 32 bit applications on a 64 bit machine (think of stuff like Steam)

Reply:
> multilib is specifically for (mainly) where we want to build something twice to get a 32 bit library and a 64 bit one for supporting 32 bit applications on a 64 bit machine (think of stuff like Steam)

Got it, that's what I had in mind, I just assumed it was necessary for x86 without thinking any further.

>  If it's not needed for x86 I don't think it's useful.

On the other hand, a majority of deps in ffmpeg use `${MULTILIB_USEDEP}` so I wanted to make vmaf follow suite. What do you think ?

Reply:
It's only useful if something ends up needing it, so let's leave it out for now.

Reply:
Okay so it `ffmpeg` fails to emerge if `libvmaf` isn't installed in a multilib scope :/ I think it's just simpler to keep the multilib ebuild than tyring to figure out how to make `ffmpeg` work without.